python_practice/calculator/simple_calc.py

```diff
@@ -0,0 +1,123 @@
+"""
+This program will be a simple calculator (add, subtract, divide, multiply)
+"""
+
+
+def add(
+        x: int,
+        y: int,
+) -> int:
+    """
+    This function will be the add operator
+
+    :param x: x will act as the first addend
+    :param y: y will act as the second addend
+    :return int: the sum will be an integer
+    """
+
+    total = x + y
+
+    return total
+
+
+def subtract(
+       x: int,
+       y: int,
+) -> int:
+    """
+    This function will be the subtraction operator
+
+    :param x will be the minuend
+    :param y will be the subtrahend
+    :return int: the difference will be an integer
+    """
+
+    difference = x - y
+
+    return difference
+
+
+def divide(
+        x: int,
+        y: int,
+) -> int:
+    """
+    This function will be the division operator
+
+    :param x will be the divisor
+    :param y will be the divident
+    :return int: the quotient will be an int with a remainder
+    """
+
+    quotient = x / y
+
+    if y == 0:
+        print('undefined')
+
+    return quotient
+
+    remainder = x % y
```

Comment:
You've already returned, none of this will execute. If you want to return both, you can use `return (quotient, remainder)` which is a tuple in python.